misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/AnyAssertionsSpec.kt

```diff
@@ -746,6 +745,36 @@ abstract class AnyAssertionsSpec(
         }
     }
 
+    prefixedDescribe("because") {
+        it("the test on the supplied subject is not throwing an assertion error") {
+            expect("filename")
+                .because("? is not allowed in file names on Windows") {
+                    containsNot("?")
+                }
+        }
+
+        it("provoke the failing of one assertion") {
+            expect {
+                expect("filename?")
+                    .because("? is not allowed in file names on Windows") {
+                        containsNot("?")
+                        startsWith("f")
+                    }
+            }.toThrow<AssertionError>()
+        }
+
+        it("provoke the failing of two assertions") {
+            expect {
+                expect(21)
+                    .because("we use the definition that teens are between 12 and 18 years old") {
+                        isGreaterThanOrEqual(12)
+                        isLessThan(18)
+                        isNoneOf(21)
+                    }
```

Comment:
this test case currently only tests that `isLessThan` works correctly (see above).